modules/web/themes/helium/styles.scss

```diff
@@ -26,9 +27,9 @@
   @include com_haulmont_sample_petclinic-helium-ext;
 
 
-  @include calendar-event(event-red, #e00000, #f4c8c8);
-  @include calendar-event(event-green, #3c853a, #c8f4ca);
-  @include calendar-event(event-yellow, #7f7e76, #f4efc8);
-  @include calendar-event(event-purple, #ffffff, #b411dd);
-  @include calendar-event(event-blue, #00a2d9, #bae0f3);
+  @include calendar-event(event-red, #DF6679, #FFC8D0);
```

Comment:
The `styles.scss` mustn't include any customization.

Reply:
done 👍 . Can you elaborate on why this is a problem? At least it worked.

Reply:
Even though it works in the current project, this is bad practice. As you know, any CUBA app can be used as an app component. Since the `styles.scss` file located in the theme root directory, it's overridden in the current application, so that any styles added to this file are lost then the app is included as an app component. For example, I included Petclinic as an app component to another app, as a result, no **dark** preset is available.

<img width="1662" alt="Screenshot 2020-05-14 at 10 29 19" src="https://user-images.githubusercontent.com/6077149/81900457-e15ca180-95cd-11ea-8595-cf285e4e413c.png">

Reply:
ok I see. This makes sense. Can you provide a PR for fixing the issue with the `dark` mode for the https://github.com/cuba-platform/cuba-petclinic/tree/2.x branch?

Reply:
Sure